packages/esm-service-queues-app/src/patient-queue-metrics/clinic-metrics.component.tsx

```diff
@@ -40,7 +38,11 @@ function ClinicMetrics() {
       return true;
     }
   });
-  const { visitQueueEntriesCount } = useVisitQueueEntries(currentServiceName, currentQueueLocation);
+  const { queueEntries, totalCount } = useQueueEntries({
```

Comment:
So I think it would be more efficient to use the `queue-entry-metric?metric=count` endpoint to get the `totalCount` if you don't actually need the queueEntries (unless we are already retrieving them on the same page and they are cached in swr anyway).

Reply:
Yeah, they are being retrieved with swr elsewhere already.